app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">What you'll need to become a teacher</h2>
+    <p>You’ll need a degree and <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a> to work in the majority of schools in England. This  includes state maintained primary, secondary and special schools.</p>
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to get QTS.</p>
+    <p>It doesn't matter when you graduated, and your degree subject doesn't necessarily need to be the subject you want to teach. Once you have QTS you could teach various subjects, providing you can show you have enough knowledge.</p>
```

Comment:
```suggestion
    <p>It does not matter when you graduated, and your degree subject does not necessarily need to be in the subject you want to teach. Once you have QTS, you can also teach any other subject, as long as you can show you have enough expertise.</p>
```

Reply:
Have changed this so it matches what we say on the subject pages. We can't use the same partial though as it talks about teaching different ages etc